docs/src/main/sphinx/release/release-409.md

```diff
@@ -0,0 +1,72 @@
+# Release 409 (1 Mar 2023)
+
+## General
+
+* Add support for dropping fields with a `DROP COLUMN` statement. ({issue}`15975`)
```

Comment:
why fields? .. isn;t columns?

Reply:
https://github.com/trinodb/trino/issues/15975 - it's nested fields, we support DROP COLUMN already.